.env.json.template

```diff
@@ -0,0 +1,5 @@
+{    # 1. Copy this file and rename it `.env`.
+    # 2. Copy your API key from https://www.weatherapi.com/
+    # 3. Paste it below, instead of yourKey
+    "WEATHER_API_KEY":"yourKey"
+}
```

Comment:
Formatting here looks wrong, could you fix this?

Reply:
At the end of the day this conflicts with the `main` branch so it really doesn't make much difference, so scrap that